torch/_dynamo/variables/torch.py

```diff
@@ -284,6 +284,11 @@ def call_function(
                     **{k: v.as_python_constant() for k, v in kwargs.items()},
                 ),
             )
+        elif self.value == torch._dynamo.external_utils.is_fullgraph_compiling:
+            # See: https://github.com/pytorch/pytorch/issues/110765
+            tx.mark_inconsistent_side_effects()
```

Comment:
@jon-chuang I am not sure whether this is necessary here?

Reply:
If you look into the original issue, we simply are committing to capturing the graph. Is that a problem for what you're trying to achieve?

Reply:
It's necessary for capturing side-effect only code. If your code has torch operation on tensor input, it has makes no difference.